README.md

```diff
@@ -16,6 +16,35 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+Here is the matrix that describes the features supported by the different
```

Comment:
Switch features and the CSI version tables below. IOW, let's introduce the versions and then features in each version.

Reply:
Thats how the initial version was, and reviewers suggested that the number of columns will keep growing as and when a new release happens.